internal/backend/sftp/foreground_solaris.go

```diff
@@ -0,0 +1,15 @@
+// +build !windows
```

Comment:
`node_solaris.go` below doesn't have this constraint, why is it needed here?

Reply:
Good point; it's not needed. I've pushed a fixed file.